src/index.ts

```diff
@@ -51,13 +52,14 @@ export const basename: typeof path.basename = function (p, ext) {
 }
 
 // parse
+// TODO
 export const parse: typeof path.parse = function (p) {
-  return path.posix.parse(normalizeWindowsPath(p))
+  return path.parse(normalizeWindowsPath(p))
 }
 
 // toNamespacedPath
 export const toNamespacedPath: typeof path.toNamespacedPath = function (p) {
-  return p
+  return normalizeWindowsPath(p)
```

Comment:
This is also a platform-dependent API that POSIX behavior is to return input as-is....